### PR TITLE
Create sharmrb.json

### PR DESCRIPTION
**Pull request by sharmrb**

### CODE REVIEW

File: _data/semesters/Summer 20/sharmrb.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ {
+       "emoji": ":sunglasses:",
+       "role": "Student",
+       "introduction": Hi My name is Rishabh , cant wait to learn more",
```
Comment (Ygilany) [inline _data/semesters/Summer 20/sharmrb.json:4]: you're missing a `"` before your introduction string

File: _data/semesters/Summer 20/sharmrb.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+       "achievements": [
+           "Got a 3.9 gpa for first year and i thinks its pretty good !",
+           
+       ],
+       "links": {
+           "site": "https://it3049c-summer20.github.io/1-online-portfolio-sharmrb/",
+           "linkedin": https://www.linkedin.com/in/rishabh-sharma-300415194/
```
Comment (Ygilany) [inline _data/semesters/Summer 20/sharmrb.json:11]: you're missing the quotations around your linkedin link here

File: _data/semesters/Summer 20/sharmrb.json (hunk 1 above)
Comment (sharmrb) [reply _data/semesters/Summer 20/sharmrb.json:4]: I will fix that now

File: _data/semesters/Summer 20/sharmrb.json (hunk 2 above)
Comment (sharmrb) [reply _data/semesters/Summer 20/sharmrb.json:11]: I will fix that now

**Pull request merged**